Datahub.Core/Components/TopBar/Profile.razor

```diff
@@ -36,9 +36,12 @@
                     </CardHeaderActions>
                 </MudCardHeader>
                 <MudCardContent>
-                    <MudText>
-                        @Localizer.GetString("You have used {0} of your allocated {1}", DatahubTools.BytesToString(_totalSpaceUsed), _maxAllocatedSpace)
-                    </MudText>
+                    <MudStack>
+                        <MudLink Href="/profile">@Localizer["View your profile"]</MudLink>
+                        <MudText>
+                            @Localizer.GetString("You have used {0} of your allocated {1}", DatahubTools.BytesToString(_totalSpaceUsed), _maxAllocatedSpace)
+                        </MudText>
+                    </MudStack>
```

Comment:
This allocation is legacy from Todd's design. we probably do not need it anymore. or atleast, not until we actually restrict and track usage